docs/events.yaml

```diff
@@ -420,14 +420,14 @@ x-events:
   properties:
     cluster_id: UUID
 
-- name: image_deleted_after_ignition_update
+- name: image_deleted_because_ignition_updated
```

Comment:
the actual action here is the image deletion, so shouldn't `deleted` somehow be the last word in this name?
@avishayt WDYT?

Reply:
Sure.  So how about ignition_updated_therefore_image_deleted?

Reply:
done